bpf_helpers.h

```diff
@@ -0,0 +1,141 @@
+#ifndef __BPF_HELPERS_H
```

Comment:
With bcc, the bpf helper functions are automatically included, but without bcc we need to do that manually.